### ISSUE TITLE
Formulating the Hypothesis

### ISSUE THREAD

**Issue by Seulah-K** [Seulah-K/statistics_study#1]
*Summarized the lecture 'Inferential and Predictive Statistics for Business' from the University of Illinois Urbana-Champaign on Coursera.

Identifying and studying problems is the first step to make the business thrive. Statistics are useful methods to help us diagnose the critical situation and improve it. 

First and foremost, we need to define what we want to seek an answer. This process is about formulating a hypothesis. 

- Null hypothesis (H0) : current belief (believed to be true)

- Alternate hypothesis (Ha or H1) : everything else that can happen

They are mutually exclusive so that cover all possibilities.

> One-Tail Test

The lecturer gives an example of the student loan.
Let's say the average amount of debt for graduates is more than $35,000. This is a common belief at this time.
Good Deal University, however, believes that their financial aid contributed to lower the amount of debt for students below $35,000.
If the university wants to prove its claim, they need to provide actual data to debunk the common belief.

- Null hypothesis (H0) : average debt ≥ $35,000

- Alternate hypothesis (Ha or H1) : average debt ＜ $35,000

To sum up, in the case of a one-tail test, the null hypothesis has any value of a parameter larger or smaller than a specified value.
(Coursera lecture note 6:44)

> Two-Tail Test 

A chocolate manufacturing plant maintains the weight of its product at 300 grams.
The quality control team takes random fifty samples to inspect whether the weight of each is equal to 300 grams. 

Here,

- Null hypothesis (H0) : weight = 300 grams

- Alternate hypothesis (Ha or H1) : weight ≠ 300 grams

We call this two-tail test, of which the null hypothesis has a specific value for the population parameter.
